### PR TITLE
Issue #500: Document disabling Pantheon integrated composer and autopilot.

### PR DESCRIPTION
**Pull request by quicksketch**
Fixes https://github.com/Lullabot/drainpipe/issues/500.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -400,6 +400,26 @@ includes:
 
 See below for CI specific integrations for hosting providers.
 
+When using Pantheon with Drainpipe, the Pantheon site should be configured to
+override some of Pantheon's default behaviors. Because Drainpipe installs
+composer dependencies, Pantheon's (Integrated Composer)[https://docs.pantheon.io/guides/integrated-composer]
```
Comment (quicksketch) [inline README.md:405]:
```suggestion
composer dependencies, Pantheon's [Integrated Composer](https://docs.pantheon.io/guides/integrated-composer)
```

**Comment by davereid** [comment]
Shouldn't the pantheon.yml be scaffolded by Drainpipe, which already sets this: https://github.com/Lullabot/drainpipe/blob/main/scaffold/pantheon/pantheon.yml

**Comment by quicksketch** [comment]
I started with a fresh install of Drupal 10 and followed the README for adding Drainpipe, and no `pantheon.yml` file was added. Maybe this is similar to https://github.com/Lullabot/drainpipe/issues/499, where the general Pantheon support _on GitHub_ has been broken. Seems like it might only be working for GitLab.

**Comment by justafish** [comment]
Thanks!

**Pull request merged**